NEWS.md

```diff
@@ -1,7 +1,12 @@
 # recipes (development version)
 
+## Improvements and Other Changes
+
 * Improved the efficiency of computations for the Box-Cox transformation (#820).
 
+* The recipes steps in this package are now tolerant to empty selections. For example, if a previous step removed all of the columns need for a later step, the recipe does not fail when it is estimated (with the exception of `step_mutate()`). The documentation in `?selections` has been updated with advice for writing selectors when filtering steps are used. (#813) 
```

Comment:
There was a bullet for this in the empty selection PR. I think when you merged it into master, it got merged into the 0.1.17 bullets

i.e. look at the first bullet here:
https://github.com/tidymodels/recipes/blob/master/NEWS.md#improvements-and-other-changes

You may want to remove that bullet or reuse it here